src/reducers/content.js

```diff
@@ -19,6 +21,14 @@ export default (state = initialState, action) => {
           [action.payload.content.data.id]: action.payload.content,
         },
       };
+    case CONTENT_ADD_CHANGE:
```

Comment:
For restoring node edit forms we should store it by entity ID/UUID instead of by bundle. That's the reason there is a dedicated action for adds, but sure this could be unified